api/institutions/views.py

```diff
@@ -1,11 +1,14 @@
 from rest_framework import generics
 from rest_framework import permissions as drf_permissions
+from rest_framework import status
+from rest_framework.response import Response
 
 from modularodm import Q
 
 from framework.auth.oauth_scopes import CoreScopes
 
-from website.models import Institution, Node, User
+from website.models import Node, User
+from website.project.model import Institution
```

Comment:
yep, ` website.models ` is reserved for StoredObjects, as this is where modularodm looks for them (I think?)